client/src/pages/SearchResult/SingleSearchResult.tsx

```diff
@@ -47,6 +54,8 @@ export function SingleSearchResultPage() {
       try {
         const data = await getRestaurant(place_id);
         setRestaurantData(data.result);
+        setPrice(data.result.price_level);
+        // add heart api later
```

Comment:
Please add a TODO to the beginning of a comment like this so make it easier to find these types of comments later through search.